joomla/joomla-cms
# Smart Search - move Filters (3x) from left to [Search Tools]

This PR moves the Filter option of all three views of **Smart Search** to the [Search Tools] button.
(See Issue #6941 regarding the inconsistency with Filter options).
## Test instructions

You have to enable the **Content - Smart Search** Plugin, and **Index your test website**
Please check the 3 different views before & after the patch: 
test all the filters and search box, clear results, choose max amount of results on a page.
## Smart Search: Manage Indexed Content
### Check the current position of the Filters

In back-end > Components > Smart Search > Indexed Content
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/9b6cf2caa2a5ca839c81409974b03437.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/741fb1b5b331b553bdbfc926edef7195.png)
## Smart Search: Manage Content Maps
### Check the current position of the Filters

In back-end > Components > Smart Search > Content Maps
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/7f31ad619fce230e4e1aaf825d1847f0.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/f2ed1d2e47e52ba1af0f66195a957e5f.png)
## Smart Search: Manage Search Filters
### Check the current position of the Filters

In back-end > Components > Smart Search > Search Filters
The Filters are on the left hand side

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/a2f3a221a31b8072071682887eabcefb.png)
### This PR changes moves the Filter

from the left to the middle column and will be visible when you click on the [Search Tools] button.

![screen shot 2015-05-15 at 14 22 52](http://issues.joomla.org/uploads/1/5314254c1b4b1a82f10485c12d4bff8d.png)
## Note

To Clear the Filters / Search box, you can use the [Clear] button.
However, sometimes you have to press 2 times to clear the results.
I don't know the cause and how to solve it...
